app/javascript/components/tree-view/reducers/index.js

```diff
@@ -0,0 +1,24 @@
+import { ActionTypes } from '@manageiq/react-ui-components/dist/wooden-tree';
+import onCheck from './oncheck';
+import onClick from './onclick';
+import { checkAll, checkSilent } from './others';
+
+export const ACTIONS = {
+  ...ActionTypes,
+  EMPTY_TREE: 'tree.empty',
+  CHECK_ALL: 'tree.checkAll',
+  CHECKED_SILENT: 'node.checked.silent',
```

Comment:
Again, this should be in the present, not the past. You're checking it now.

Reply:
And let's use the `tree` prefix consistently for tree actions.